.buildkite/pipelines/main/platforms/tester_linux.yml

```diff
@@ -61,6 +61,11 @@ steps:
       unset JULIA_DEPOT_PATH
       unset JULIA_PKG_SERVER
 
+      # Make sure that temp files and temp directories are created in a location that is
```

Comment:
Why is this required?

Reply:
Inside the Buildkite sandbox, the `/tmp` directory lives on a tempfs that is backed by 1GB of RAM.

On certain Buildkite jobs (e.g. when we run the Pkg test suite with the Pkg server disabled), we end up cloning a lot of Git repos, and as a result we end up running out of space in the tempfs.

Reply:
Increase the size of the tempfs?

Reply:
We could do that. I know that @staticfloat and @maleadt have expressed concerns about that in the past. We have multiple Buildkite agents on each machine, and we don't want to use up too much RAM.